rpcs3/Emu/RSX/VK/vkutils/device.cpp

```diff
@@ -606,6 +613,13 @@ namespace vk
 			enabled_features.logicOp = VK_FALSE;
 		}
 
+		if (!pgpu->features.textureCompressionBC)
```

Comment:
We should add `&& pgpu->driver_vendor() == v3dv` here. Let other unknown drivers fail since we don't know how they'll behave.

Reply:
Done